packages/material-ui/src/CssBaseline/CssBaseline.js

```diff
@@ -34,9 +34,9 @@ const useStyles = makeStyles(
 /**
  * Kickstart an elegant, consistent, and simple baseline to build upon.
  */
-function CssBaseline(props) {
+function CssBaseline({ children = null }) {
```

Comment:
Is this meant to be destructured here? I know it's only one prop so maybe it's not a big deal.

Reply:
Probably added this to test react-docgen. Will remove.

Reply:
>  I know it's only one prop so maybe it's not a big deal.

Well actually...destructuring in the function parameter is es6 while base object destrcuturing is es5. So there is an implication for required babel plugins.